tools/docker/tests/test_flows.sh

```diff
@@ -148,9 +148,7 @@ test_client_steering_mandate() {
     sleep 1
     dbg "Confirming Client Steering Request message was received - mandate"
     check docker exec -it repeater1 sh -c \
-        'grep -i -q "CLIENT_STEERING_REQUEST_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
-    check docker exec -it repeater1 sh -c \
-        'grep -i -q "CLIENT_STEERING_REQUEST_MESSAGE" /tmp/$USER/beerocks/logs/beerocks_agent_wlan2.log'
+        'grep -i -q "Got steer request" /tmp/$USER/beerocks/logs/beerocks_agent_wlan0.log'
```

Comment:
This change would ideally have been in a separate commit, together with adding this log message to bwl. But it's not important enough to still do it now. (nitpick)